code/analyze/ac/plots/de_PlotFFTs.m

```diff
@@ -309,18 +309,30 @@
           pwrLH = squeeze(ffts.orig2D(1,1,:,:)).*(pd<0).*(ffts.pals.an2D<=0.05);
 
           imgRH = ifft2(pwrRH); imgRH=imgRH(1:mSets.nInput(1), 1:mSets.nInput(2));
+          rMin = min(min(imgRH));
+          rMax = max(max(imgRH));
+          imgRH = (imgRH - rMin) / (rMax-rMin);
+          
           imgLH = ifft2(pwrLH); imgLH=imgLH(1:mSets.nInput(1), 1:mSets.nInput(2));
+          lMin = min(min(imgLH));
+          lMax = max(max(imgLH));
+          imgLH = (imgLH - lMin) / (lMax-lMin);
 
+
+          % Now normalize the images so they are properly visible
           % Now take an image, and show.
+         
 
           subplot(1,2,1); colormap gray;
           title(sprintf('LH (\\sigma=%3.2f) p<0.05 recon', mSets.sigma(end)));
           imagesc(imgLH, [0 1]);
+          ylabel('Normalized')
```

Comment:
Well it's the pixel values that are normalized. But '''Normalized pixel''' should work?